generate_zeebe_initial_contact.py

```diff
@@ -0,0 +1,28 @@
+""""This script generates the initial contact string for Zeebe clusters in a multi-region setup."""
+
+# For now the script is focused on dual-region.
+# It will be extended if anything specific changes for more than 2 regions.
+
+def generate_string(ns_0, ns_1, release, count):
+    """Generates the initial contact string for Zeebe clusters in a multi-region setup."""
+    result = ""
+    for i in range(count // 2):
+        result += f"{release}-zeebe-{i}.{release}-zeebe.{ns_0}.svc.cluster.local:26502,"
+        result += f"{release}-zeebe-{i}.{release}-zeebe.{ns_1}.svc.cluster.local:26502,"
+    return result
+
+# Taking inputs from the user
+namespace_0 = input("Enter a namespace for cluster in region 0: ")
+namespace_1 = input("Enter a namespace for cluster in region 1: ")
+helm_release_name = input("Enter Helm release name: ")
+cluster_size = int(input("Enter Zeebe cluster size: "))
+
+if cluster_size % 2 != 0:
```

Comment:
I agree, smaller than 4 doesn't make much sense since 3 isn't supported and a cluster of 2 will be difficult with the partitions and generally questionable.